pstress/mysqld_options_ms_80.txt

```diff
@@ -473,10 +499,23 @@
 --ft-query-expansion-limit=2047
 --ft-query-expansion-limit=-1125899906842624
 --ft-query-expansion-limit=1125899906842624
+--ft-query-extra-word-chars=1
+--ft-query-extra-word-chars=0
 --gdb=1
 --gdb=0
 --general-log=1
```

Comment:
Exclude this. We dont want to disable general logs

Reply:
excluded general log